.ci/cico_common.sh

```diff
@@ -68,6 +68,8 @@ function installAndStartMinishift() {
   oc adm policy add-cluster-role-to-user cluster-admin developer
   oc login -u developer -p developer
 
+  # shellcheck source=che-cert_generation.sh
+  # shellcheck disable=SC1091
```

Comment:
hello, why we disable check ?

Reply:
because we know where the file is and we don't need to worry about it moving suddenly.

Reply:
and every attempt I did to make this check happy failed. so... https://github.com/koalaman/shellcheck/wiki/SC1091 decided to disable it.